screwdriver/screwdriver.go

```diff
@@ -110,8 +110,9 @@ type Job struct {
 
 // CommandDef is the definition of a single executable command.
 type CommandDef struct {
-	Name string `json:"name"`
-	Cmd  string `json:"command"`
+	Name      string `json:"name"`
+	Cmd       string `json:"command"`
+	AlwaysRun string `json:"alwaysRun"`
```

Comment:
I don't think so. The `permutations` is for the `matrix` field:
```
const matrix = Hoek.reach(doc, `jobs.${jobName}.matrix`, {
    default: {}
});
const permutations = keymbinatorial(matrix);
```

You can try running this:
https://gist.github.com/d2lam/f7b39753aa33b9ceac7327ed11a13243

It's showing that `alwaysRun` value is `false` as boolean